example/src/components/ChatboxWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC } from 'react';
+import { List } from "immutable";
+import Chatbox, { ChatMessage, PartialChatMessage, ImmutableMember } from "@graasp/chatbox";
+import { MUTATION_KEYS } from '@graasp/query-client';
+import { useMutation, hooks } from '../config/queryClient';
+
+type Props = {};
+
+const ChatboxWrapper: FC<Props> = () => {
+  // this is the id of the item to which the chat is attached (folder, document ...)
+  const chatId = '39370f67-2153-4ab9-9679-b1966542d27d';
+  // todo: make member dynamic with useMember hook
+  const member = new ImmutableMember({
+    name: 'basile',
+    id: 'a1112eb7-7f28-4fdb-92cc-28171177463f',
+  });
+  // todo: fetch other members using hook
+  const members = [
+    member,
+    {
+      name: 'bob',
+      id: 'bob-id',
+    },
+  ];
```

Comment:
Maybe let's factor out in a `fixtures` file. Use the member mutation if you think it fits, I don't mind having a mock member if it works.

Reply:
For this i tried using the `useMembers` hook but i has problems with `crypto` package. I think for the moment i will leave it like this but the idea is to have the example project work with the local backend and the tests work with the fixtures.